tests/testthat/test-metadata.R

```diff
@@ -1,3 +1,57 @@
+test_that("cache_data runs successfully", {
+  # Don't skip on CI
+  if (Sys.getenv("GITHUB_ACTIONS") == "true") {
+    cli::cli_alert_info("Running cache_data() in CI...")
+  } else {
+    skip_on_cran()
+  }
+
+  cache_folder <- get_cache_folder()
+  cli::cli_alert_info("Cache folder path: {cache_folder}")
+
+  expect_no_error(cache_data())
+
+  # Check that a file exists in cache (if not in CI)
+  if (Sys.getenv("GITHUB_ACTIONS") != "true") {
+    cached_files <- files_to_cache()
+    existing_files <- list.files(cache_folder, full.names = TRUE)
+    expect_true(any(basename(existing_files) %in% cached_files),
+                info = "At least one expected data file should exist in the cache"
+    )
+  }
+})
+
+test_that("load_sql_data runs successfully", {
+  # Don't skip on CI
+  if (Sys.getenv("GITHUB_ACTIONS") == "true") {
+    cli::cli_alert_info("Running load_sql_data() in CI...")
+  } else {
+    skip_on_cran()
+  }
+  # Changed to expect_no_error becasue load_sql_data throws warnings about pkg versions
+  db_path <- surveyjoin:::sql_folder()
+  cli::cli_alert_info("Database path: {db_path}")
+
+  load_sql_data()
+
+  # Check if database was created
+  db_path <- sql_folder()
+  expect_true(file.exists(db_path), info = paste("Database file should exist at:", db_path))
+
+  # test that SQLite database contains data
+  g <- get_data(regions="pbs")
+  expect_gt(nrow(g), 300000)
+
+  # test data versioning
+  # ver <- data_version()
+  # expect_equal(nrow(ver), 6L)
+  # expect_equal(names(ver), c("file", "last_updated"))
+  # expect_equal(ver$file, c("pbs-catch.rds", "pbs-haul.rds",
+  #                          "afsc-catch.rds", "afsc-haul.rds",
+  #                          "nwfsc-catch.rds", "nwfsc-haul.rds"))
```

Comment:
Was there something wrong with this code that needs fixed or should it just be deleted?